shopinvader_product_template_multi_link/models/shopinvader_product.py

```diff
@@ -50,7 +50,9 @@ def _get_product_links_by_type(self, links):
             grouped[code] |= link
         res = {}
         for code, links in grouped.items():
-            res[code] = [self._get_product_link_data(x) for x in links]
+            res[code] = [
```

Comment:
@simahawk In case of product links that are not bound to shopinvader backend, we can have void dict values.

e.g.: 'up_selling': [{}, {},..., {'id': id}]

This commit can be forwarported IMHO.